ssl-config/src/main/java/com/palantir/remoting/ssl/KeyStoreConfiguration.java

```diff
@@ -33,7 +32,7 @@
 public abstract class KeyStoreConfiguration {
 
     @Value.Parameter
-    public abstract Path path();
+    public abstract URI path();
```

Comment:
To clarify: the intent of this change is that a user can pass in an arbitrary URI and the app can call `uri.toURL().openStream()` or `Paths.get(uri)`, assuming they have a handler registered for the URI scheme passed in. Correct?